lib/plugins/bullMqMetricsPlugin.spec.ts

```diff
@@ -161,9 +161,25 @@ describe('bullMqMetricsPlugin', () => {
     })
 
     await processor.spy.waitForJobWithId(jobId, 'completed')
-    await setTimeout(200)
 
-    const responseAfter = await getMetrics()
+    const responseAfter = await vi.waitUntil(
```

Comment:
changes unrelated to the PR, but for whatever reason test is failing due to data generation taking longer than before. I assume GA is being slow.
locally it was passing without any changes